src/js/camresolutionstest.js

```diff
@@ -144,9 +144,9 @@ CamResolutionsTest.prototype = {
     video.setAttribute('muted', '');
     video.width = resolution[0];
     video.height = resolution[1];
-    attachMediaStream(video, stream);
+    video.srcObject = stream;
```

Comment:
Since it's deprecated, we should be able to use srcObject on Chrome and Firefox.